### PR TITLE
Right rail improvements

### PR DESCRIPTION
**Pull request by jerelmiller**
Closes https://github.com/newrelic/docs-website/issues/847
Closes https://github.com/newrelic/docs-website/issues/750
Closes https://github.com/newrelic/docs-website/issues/741
Relates to https://github.com/newrelic/docs-website/issues/427

## Description

This PR addresses several items:

* Updates the design of the right rail to conform to recent feedback
* Adds a `GitHubIssueButton` component to make it easy to create links to issues in GitHub
* Adds a `CreateIssueButton` that wraps `GitHubIssueButton` to maintain consistency for GH buttons to report issues
* Include environment information for the user in the GitHub issue body by default

## Screenshots

Screen Shot 2021-02-12 at 1 46 21 AM
Screen Shot 2021-02-12 at 1 46 29 AM

### CODE REVIEW

**Comment by jpvajda** [comment]
Love it... :shipit:

File: packages/gatsby-theme-newrelic/src/components/GitHubIssueButton.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+* ${getEnvironmentString('OS', os.name, os.version)}
+* ${getEnvironmentString('Browser', browser.name, browser.version)}
+* ${getEnvironmentString(
+    'Device',
+    device.type && `[${device.type}]`,
+    device.vendor,
+    device.model
+  )}
+`;
+};
```
Comment (zstix) [inline packages/gatsby-theme-newrelic/src/components/GitHubIssueButton.js:30]: Oooh this is such a good idea!

**Comment by austin-schaefer** [comment]
Looking so good! Great work @jerelmiller @djsauble @jpvajda

**Review by zstix — APPROVED** [review]
This looks great. Thanks for the refactor!

File: packages/gatsby-theme-newrelic/src/components/GitHubIssueButton.js (hunk 1 above)
Comment (jerelmiller) [reply packages/gatsby-theme-newrelic/src/components/GitHubIssueButton.js:30]: Thank you!

**Comment by jerelmiller** [comment]
Thanks so much @austin-schaefer!!!

**Pull request merged**